OpenEdXMobile/res/values/strings.xml

```diff
@@ -469,9 +469,9 @@
     <!-- Prompt leading user to create account screen -->
     <string name="sign_up_and_learn">Sign up and start learning</string>
     <!-- In Sign up screen -->
-    <string name="sign_up_with">Sign up with</string>
+    <string name="sign_up_with">Register with</string>
     <!-- In Sign up screen -->
```

Comment:
What's docstring? I looked in confluence and couldn't find anything. Is it somewhere else?